packages/application-extension/style/base.css

```diff
@@ -8,7 +8,12 @@
   width: calc(var(--jp-private-sidebar-tab-width) + var(--jp-border-width));
 }
 
-#jp-topbar-spacer {
-  flex-grow: 1;
-  flex-shrink: 1;
+#jp-top-bar {
+  --jp-private-toolbar-height: var(--jp-private-menu-panel-height);
```

Comment:
Should this be moved to a `:root` definition, similar to other private variables like:

https://github.com/jupyterlab/jupyterlab/blob/ce09e0daabd48e4e90c13234137cbe4c6783eea0/packages/application/style/menus.css#L10-L13

Reply:
I did not move those because we don't want to modify all the styles but only overwrite the default for that specific toolbar.